Library/Homebrew/test/cmd/cleanup_spec.rb

```diff
@@ -7,6 +7,17 @@
 end
 
 describe "brew cleanup", :integration_test do
+  describe "--prune=0" do
```

Comment:
Definitely don't want a new integration test for these as they are super slow.

Reply:
Removed `--prune=0`. What about `--prune=garbage` test below -- keep or yank?

Reply:
Kill both, thanks. Only want one integration test per command.

Reply:
Done! I also squashed the commits.